### ISSUE TITLE
In order to [easily know what client an order comes from] As a [admin] I want [see the client's name]

### ISSUE THREAD

**Issue by AdrienSICKLO** [coopcycle/coopcycle-web#3895]
Right now, on the orders' list, there is no easy way to match an order to a store. In fact, it does not appear in the list, nor does it appear if you click on the order's link. 

We should add another column which should be "Client" and change the current "Client " column as "User". 

![image](https://github.com/coopcycle/coopcycle-web/assets/59842043/2e4c2053-3dc3-4a5b-ba41-2761c4fb80b4)

![image](https://github.com/coopcycle/coopcycle-web/assets/59842043/1a0867b6-8edb-41ae-99e4-7c538adc28e6)

**Comment by vladimir-8** [comment]
@AdrienSICKLO just to clarify, by "Client" do you mean the name of the "store"?

**Comment by AdrienSICKLO** [comment]
Yes if existing (cuz food order wouldn't be a store but would still create an order)

**Comment by Paul-Eraman-CoopCycle** [comment]
in the new designs this is part of it, but if its easy to make that store/shop name a url link to their store/shop page, i dont see why not do it now

**Comment by alexsegura** [comment]
Fixed in 20c9c362c9c6b852bbc2484afc5f89c8f2bcc38e
